website/docs/addon_unreal_admin.md

```diff
@@ -94,7 +94,7 @@ pip.main(["install", "pyside6"])
 Be aware that calling **pip** like so is deprecated by pip itself and might not work in newer versions.
 
 :::note
-For Unreal Engine versions `>= 5.4`, use `pyside6`. For earlier versions, use `pyside2`.
+For Unreal Engine versions `>= 5.4`, use `pyside6` with the version not later than *6.8.0*. For earlier versions, use `pyside2`.
```

Comment:
I wonder if it should be rephrased to `6.7.x` instead.
Maybe @BigRoy can provide more useful comment.

Reply:
I actually do not think the issue is necessarily with 6.8.0+ but it may also only be with 6.8.0 only (which was the latest release at the time). There are now two more patch releases since then which I suspect won't fix it yet, but they might have already?

"not later than 6.8.0" also sounds like 6.8.0 itself is fine, which it isn't. So I'd definitely rephrase this.

So this is indeed fine:
```suggestion
For Unreal Engine versions `>= 5.4`, use `pyside6` with version *6.7.x*. For earlier versions, use `pyside2`.
```

However, I wonder if it warrants to explain why one should not use 6.8.0. It could be in a separate note from this one:

```md
Early releases of PySide 6.8.x has shown issues/crashes. As such, we recommend using 6.7.x releases for the time being in the hope that future releases of 6.8 become more stable.
```

Reply:
We can add it as remark.